lib/hg-utils.coffee

```diff
@@ -242,7 +242,11 @@ class Repository
       params = []
     if !util.isArray(params)
       params = [params]
-    child = spawnSync('hg', params)
+
+    return '' unless @isCommandForRepo(params)
+
+    options = { cwd: @rootPath }
+    child = spawnSync('hg', params, options)
```

Comment:
tree-view file diff/status stopped working here. Removed cwd options and it went fine. Do we really need cwd option since we're always passing the repository/file path to hg commands? 

I've also release v2.0.3 with your fixes :) 

Thanks for your support. I'll add a contributors section on readme.md and put your name/github account there.